theme/snippets/news-feed.liquid

```diff
@@ -7,7 +8,16 @@
     {% for thumbnail in multi_thumbnails %} 
       {% assign current_thumbnail = thumbnail %}
     {% endfor %}
-    {% render 'news-feed-item', cloudinary_src: current_thumbnail.cloudinary_src, image_alt: current_thumbnail.alt, body: value, cta_label: shop.metafields.accentuate.news_feed_item_cta_label[forloop.index0], link: shop.metafields.accentuate.news_feed_item_link[forloop.index0] %}
+    {% comment %}
+    On the homepage, hide news feed items with happens_at dates in the past. On the news feed page, show all news feed items. 
+    {% endcomment %}
+    {% if request.path == '/' %} 
+      {% if shop.metafields.accentuate.news_feed_item_happens_at[forloop.index0] >= today %}    
+        {% render 'news-feed-item', cloudinary_src: current_thumbnail.cloudinary_src, image_alt: current_thumbnail.alt, body: value, cta_label: shop.metafields.accentuate.news_feed_item_cta_label[forloop.index0], link: shop.metafields.accentuate.news_feed_item_link[forloop.index0] %}
+      {% endif %}
+      {% else %}
+        {% render 'news-feed-item', cloudinary_src: current_thumbnail.cloudinary_src, image_alt: current_thumbnail.alt, body: value, cta_label: shop.metafields.accentuate.news_feed_item_cta_label[forloop.index0], link: shop.metafields.accentuate.news_feed_item_link[forloop.index0] %}
+    {% endif %}
```

Comment:
On the homepage (`/`) we only render a news feed item if today's date is >= the news feed `happens_at` date. 

For all other pages, render the news feed item as usual.